README.md

```diff
@@ -1,4 +1,40 @@
-# RoBorregos @Home
-[master/README.md](https://github.com/RoBorregos/Robocup-Home/blob/master/README.md)
+<img src="logo_negro.png" width="400" ><img> 
 
-Official documentation of the project found in the [Wiki](https://github.com/RoBorregos/Robocup-Home/wiki)
\ No newline at end of file
+# RoBorregos Robocup-Home
+This repository contains the development of RoBorregos' robotic solution for [RoboCup's @HOME competition](https://athome.robocup.org/), Open Platform League (OPL), where the robot must be capable of interacting with people, manipulate objects and have intelligent responses for everyday home tasks.
+
+See [RoboCup @HOME Rules](https://robocupathome.github.io/RuleBook/rulebook/master.pdf).
+
+## Project details
+
+This project is made using:
+- [ROS Melodic](https://www.ros.org/)
+- [Rasa NLU](https://rasa.com/)
+- [Tensorflow](https://www.tensorflow.org/learn)
+- [PaddlePaddle](https://github.com/paddlepaddle/paddle)
+- [Arduino](https://www.arduino.cc/)
+- [VIA EPIA](https://www.viatech.com/en/support/eol/epia-eol/)
+- [Jetson](https://developer.nvidia.com/EMBEDDED/jetson-nano-developer-kit)
+
+<img src="SystemDesignv3.png" width="1500" ><img> 
```

Comment:
![image](https://user-images.githubusercontent.com/36351179/106523522-dec91180-64a6-11eb-80bd-796176165d46.png)

It is kind of difficult to see some things in github dark version.